ui/component/channelDiscussion/index.js

```diff
@@ -1,6 +1,7 @@
 import { connect } from 'react-redux';
 import { withRouter } from 'react-router';
 import { DISABLE_COMMENTS_TAG } from 'constants/tags';
```

Comment:
<img width="1032" alt="image" src="https://user-images.githubusercontent.com/64950861/167883507-0b41ca7f-7b9f-4ab9-a563-767c1aaadbf1.png">

- [ ] On mobile layout, I just keep making nested comments.  When it switches to thread mode, there is a memory leak warning.  I think just need to cleanup a setTimeout or something.